table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveUtils.java

```diff
@@ -165,18 +170,22 @@ public static ColumnStatisticsInfo toProto(ColumnStatisticsObj colStats) {
             .setStringStats(StringColumnStatsData.newBuilder()
                 .setNumDVs(stringData.getNumDVs()).setAvgColLen(stringData.getAvgColLen())
                 .setMaxColLen(stringData.getMaxColLen())
-                .setNumNulls(stringData.getNumNulls()).setBitVectors(stringData.getBitVectors())
+                .setNumNulls(stringData.getNumNulls())
+                .setBitVectors(stringData.getBitVectors() == null
+                    ? "" : stringData.getBitVectors())
                 .build()).build());
       }
     }
-    if (statsData.isSetStringStats()) {
+    if (statsData.isSetBinaryStats()) {
```

Comment:
maybe a different PR, but can we have unittests for these types of bugs?

Reply:
will open a different PR